packages/stream_chat/lib/src/core/api/sort_order.dart

```diff
@@ -13,6 +13,15 @@ part 'sort_order.g.dart';
 /// Example: `Sort<ChannelState>([pinnedAtSort, lastMessageAtSort])`
 typedef SortOrder<T extends ComparableFieldProvider> = List<SortOption<T>>;
 
+/// Defines how null values should be ordered in a sort operation.
+enum NullOrdering {
+  /// Null values are treated as less than any non-null value.
+  nullsFirst,
+
+  /// Null values are treated as greater than any non-null value.
+  nullsLast,
+}
```

Comment:
First and Last might be a bit confusing, because it's also depending on the direction of the sorting.

The default settings actually put it last right?
```dart
    this.direction = SortOption.DESC,
    this.nullOrdering = NullOrdering.nullsFirst,
```

Maybe `least` and `most` instead of first and last?

Reply:
I am following the naming convention used in postgres. This also matches our backend.

Reply:
[fix: fix sort option null order](https://github.com/GetStream/stream-chat-flutter/pull/2269/commits/ea548399730f8a486429bfe207f977449e0a1431)